README.md

```diff
@@ -1 +1,22 @@
-# Graph Network Simulator
\ No newline at end of file
+# Graph Network Simulator
+
+## Getting Started
+
+### Building Environment on Frontera
+
+- ssh to frontera, start an idev session on rtx node (i.e., GPU-enabled node)
+- download miniconda: `wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh`
+- install miniconda: `bash Miniconda3-latest-Linux-x86_64.sh`
+- after install start a new shell: `bash`
+- create new conda env: `conda create --name gnn python=3.8`
+- activate new env: `conda activate gnn`
+- load system modules: `module load cuda cudnn`
+- install pytorch: `conda install pytorch torchvision torchaudio cudatoolkit=11.3 -c pytorch`
+- install pyg: `conda install pyg -c pyg -c conda-forge`
```

Comment:
> GPU-enabled Frontera node

```shell
# download and install miniconda 
wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
sh ./Miniconda3-latest-Linux-x86_64.sh`

# create and activate a new conda env: 
conda create --name gns python=3.8
conda activate gns

# load CUDA modules
module load cuda cudnn

# install pytorch and pyg
conda install pytorch torchvision torchaudio cudatoolkit=11.3 -c pytorch
conda install pyg -c pyg -c conda-forge

Reply:
Added sh ./Miniconda3-latest-Linux-x86_64.sh `-b -f` to conda installer to silently install without interactions